### ISSUE TITLE
"Go to route controller" opens with improper path casing

### ISSUE THREAD

**Issue by theRealPadster** [Julien-R44/adonis-vscode-extension#19]
## Notes:
- I'm using macOS 13.1
- When I cmd+click, it opens the controller file fine (since macOS is case-insensitive), but the folder and filename casings in the breadcrumbs and tab name are incorrect. 
- I'm not sure if this is a bug with the extension or VS Code itself. 

## Package version
Using AdonisJS Extension v0.9.1

## Node.js and npm version
- node: 16.14.0
- npm: 8.3.1

## Sample Code (to reproduce the issue)
- Add `/app/Controllers/Http/StripeController.ts`
- In `/start/routes.ts`:
```ts
Route.group(() => {
  Route.post('/add-customer', 'StripeController.addCustomer').as('add-customer');
  Route.post('/payment-intent', 'StripeController.generatePaymentIntent').as('add-payment-intent');
}).prefix('/stripe').as('stripe');
```
- Then cmd + click on `StripeController.generatePaymentIntent` and it opens `/app/controllers/Http/Stripecontroller.ts`
- The proper path is `/app/Controllers/Http/StripeController.ts`

<img width="312" alt="image" src="https://user-images.githubusercontent.com/1565883/214679242-bc703ae9-3404-4f46-a2b0-c0d531c180c8.png">

<img width="448" alt="image" src="https://user-images.githubusercontent.com/1565883/214679125-09cb2e4d-cf00-4941-88d4-573debc2b763.png">

**Comment by Julien-R44** [comment]
Hey, thanks for the detailed report

I also noticed that, and will need to investigate a bit more, but looks like a bug with VSCode itself : 

https://github.com/microsoft/vscode/issues/123660
https://github.com/microsoft/vscode/issues/94307

**Comment by Julien-R44** [comment]
This should be fixed with 1.1.5 pre-release. Feel free to re-open if not
